content/docs/1.7.0/deploy/important-notes/index.md

```diff
@@ -246,3 +246,19 @@ The attribute `backendStoreDriver`, which is defined in the parameters of Storag
 ### Updating the Linux Kernel on Longhorn Nodes
 
 Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+
+### Storage Network Support for Read-Write-Many (RWX) Volumes
+
+Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, there is a limitation due to the reliance on Multus.
+
+Multus networks operate within the Kubernetes network namespace. This restricts Longhorn to mount NFS endpoints within the CSI plugin pod container network namespace. As a result, when the CSI plugin pod restart, NFS mount connections to the Share Manager pod become unresponsive. This occurs because the previous namespace, where the connection was established, is no longer available.
+
+To alleviate this, Longhorn provides two settings:
```

Comment:
```suggestion
Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, the network's reliance on Multus results in a significant restriction.

Multus networks operate within the Kubernetes network namespace, so Longhorn can mount NFS endpoints only within the CSI plugin pod container network namespace. Consequently, NFS mount connections to the Share Manager pod become unresponsive when the CSI plugin pod restarts. This occurs because the namespace in which the connection was established is no longer available.

Longhorn circumvents this restriction by providing the following settings:
```